examples/federated-learning/main.py

```diff
@@ -122,6 +134,11 @@ def build_data_pipeline(cls, local_data_file):
   # TODO: can move model_fn in here -- we leave it up to the user atm
 
 def main(_):
+  if not FLAGS.secure_aggr and FLAGS.reptile:
+    print("ERROR: Secure Aggregation must be used with "
+          "the reptile meta-learning algorithm.")
+    return
+
   if FLAGS.remote_config is not None:
     config = tfe.RemoteConfig.load(FLAGS.remote_config)
     config.connect_to_cluster()
```

Comment:
we should probably change this its automatically called by `tfe.set_config`, say abstracted under `initialize` or something

Reply:
https://github.com/tf-encrypted/tf-encrypted/issues/710